doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -217,6 +220,11 @@ numbers (see :ref:`datatypes`).
 Sorts
 -----
 
+.. index::
+   single: Set
```

Comment:
Adding this index entry reveals a bug:
The next index entry `Set (cmd)` now appears as `(cmd)`.
On the other hand, if I put `Set (sort)` here, `Set (cmd)` is displayed normally but `Set (sort)` is displayed as `(sort)`.
I have observed the same bug with the two indices for `term` (one of them being added in this PR).

@cpitclaudel @maximedenes Can you look into this?